### DISCUSSION TITLE
Can't upload into storage, can't list buckets in dashboard

### DISCUSSION THREAD

**Discussion by nmanolov** [supabase/supabase#15392] [Questions]
Hi!

I'm moving my app to a new environment and can't upload objects into storage on it.
The same code works on another supabase api url. Both DBs have been made by the same migrations.
When I try to upload I get a ```500 - Internal Server Error```.
![Screenshot 2023-06-27 at 17 29 32](https://github.com/supabase/supabase/assets/3851929/6f40528e-4f0e-4834-b4d7-37feb47d23cb)

When I tried to look it up in the dashboard, it couldn't load the buckets. It gets a 500 and shows warning.
![Screenshot 2023-06-27 at 17 28 18](https://github.com/supabase/supabase/assets/3851929/b89e3f1c-f57b-4cdd-9a3a-d0ca44c5211a)

![Screenshot 2023-06-27 at 17 19 50](https://github.com/supabase/supabase/assets/3851929/e00a6367-3d5e-4c6c-96ef-64949b588129)

I have a bucket, that I can query through the SQL editor.
![Screenshot 2023-06-27 at 17 21 31](https://github.com/supabase/supabase/assets/3851929/f38142e7-6261-4d93-9e37-118c22e7ef70)

Any pointers will be appreciated.

**Comment by nmanolov** [comment]
The storage team found that my `storage.migrations` had some irregular data in it and had it fixed.

**Reply by TheSinding** [reply]
I have the exact same issue, what was the solution?

**Comment by TheSinding** [comment]
I got it working, but i had to delete my entire Postgres Data folder (I'm self-hosting) - Not really ideal, if it was a production environment.

For anyone else coming across this issue: I guess it's something that failed duing a migration process like @nmanolov originally had.
I just didn't have the patience to wait for an answer, so I took a backup of my data, nuked the db folder and now it works.
